src/js/controllers/app.controller.js

```diff
@@ -31,6 +31,21 @@ function AppCtrl ($scope, id, net, keychain, txQueue, appManager, rpTracker,
   store.set('announcement', true);
   $scope.showAnnouncement = store.get('announcement');
 
+  // For new terms banner
+  if (store.get('terms') === false) {
```

Comment:
If the banner has already been dismissed by the user, it shouldn't show up again.